<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textConsole">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>781</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>520</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadDb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Load database</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textDb">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>491</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="loadVilkar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Load vilkar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textVilkar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>491</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="compareInit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">&quot;Text-align:left&quot;</string>
    </property>
    <property name="text">
     <string>Compare initials</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadHk">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Load HK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compareCpr">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">&quot;Text-align:left&quot;</string>
    </property>
    <property name="text">
     <string>Compare cpr</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textHk">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>491</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="loadEmails">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Emails</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compareEmails">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">&quot;Text-align:left&quot;</string>
    </property>
    <property name="text">
     <string>Compare emails</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEmails">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>491</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Database Comparer v1.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
